indico/modules/events/surveys/fields/base.py

```diff
@@ -86,6 +86,10 @@ def wtf_field_kwargs(self):
         """Returns a dict of kwargs for this field's wtforms field"""
         return {}
 
+    def get_actual_answer(self, answer):
```

Comment:
Well it still returns a python object (i.e. not a joined list of options in case of multi-select fields)..